lambda/requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
Should this have a version range?

Reply:
We can do but again, this is very unlikely to change unless Amazon change the structure of their security groups which seems unlikely. If we ever do update it, any range we put in now will be out of date.